lab_1_classify_by_unigrams/start.py

```diff
@@ -34,5 +33,6 @@ def main() -> None:
 
     assert result, "Detection result is None"
 
+
```

Comment:
revert

Reply:
Why should I do a revert? My start.py is like in start.py from repository.

Reply:
what do you say about the 36 line?